PWGJE/Tasks/hffragmentationfunction.cxx

```diff
@@ -135,15 +204,62 @@ struct HfFragmentationFunctionTask {
         registry.fill(HIST("h_d0_mass"), d0Candidate.m());
         registry.fill(HIST("h_d0_eta"), d0Candidate.eta());
         registry.fill(HIST("h_d0_phi"), d0Candidate.phi());
+
         // filling table
         distJetTable(axisDistance, jet.pt(), jet.eta(), jet.phi(), d0Candidate.pt(), d0Candidate.eta(), d0Candidate.phi(), d0Candidate.m(), d0Candidate.y());
+
         break; // get out of candidates' loop after first HF particle is found in jet
       }        // end of D0 candidates loop
 
     } // end of jets loop
 
   } // end of process function
-  PROCESS_SWITCH(HfFragmentationFunctionTask, processDataChargedSubstructure, "charged HF jet substructure", true);
+  PROCESS_SWITCH(HfFragmentationFunctionTask, processDataChargedSubstructure, "charged HF jet substructure", false);
+
+  void processMcChargedMatched(JetMcCollisions const& mccollisions,
+                               JetCollisionsMCD const& collisions,
+                               JetMCDTable const& mcdjets, JetMCPTable const& mcpjets,
+                               CandidatesD0MCD const& mcdd0cands, CandidatesD0MCP const& mcpd0cands,
+                               JetTracks const& tracks, JetParticles const& particles)
```

Comment:
It's easier to read when there is one argument per line in the process functions.

Reply:
Thanks for the suggestion Vit, just applied